SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -187,16 +206,12 @@ void FSpatialNetGUIDCache::RemoveEntityNetGUID(Worker_EntityId EntityId)
 	// Remove actor subobjects.
 	USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(Driver);
 
-	improbable::UnrealMetadata* UnrealMetadata = SpatialNetDriver->StaticComponentView->GetComponentData<improbable::UnrealMetadata>(EntityId);
-	if (UnrealMetadata == nullptr)
-	{
-		return;
-	}
+	UClass* Class = SpatialNetDriver->EntityRegistry->GetActorFromEntityId(EntityId)->GetClass();
```

Comment:
Check that actor isn't null (I'm thinking of edge cases when closing the editor etc)